src/components/PeopleFilters.tsx

```diff
@@ -33,63 +78,42 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {['16', '17', '18', '19', '20'].map((century: string) => (
+              <SearchLink
+                params={{ centuries: getCenturies(century) }}
+                className={classNames('button', 'mr-1', {
+                  'is-info': !getCenturies(century).includes(century),
```

Comment:
You are using this function twice in the JSX. Why don't you create a const for it? 
Like `const selectedCenturies = getCenturies(century)` or something.